src/Input/ToggleSwitch/ToggleSwitch.interface.ts

```diff
@@ -0,0 +1,25 @@
+interface ToggleSwitchProps {
```

Comment:
This is purely a matter of convention/personal preference. In aries, it seems that most interfaces are defined in the component that needs them. In this case `ToggleSwitchProps` belongs only to `ToggleSwitch.tsx` while the other two belong to `Styles.ts` so I think it more appropriate to just define them in their own files rather than a separate file.

It might be the case that we eventually decide that all interfaces should reside in their own file, but for now I think we can follow the convention.